src/GeneralStateTestsFiller/stExtCodeHash/extCodeHashCreatedAndDeletedAccountRecheckInOuterCallFiller.json

```diff
@@ -0,0 +1,81 @@
+{
+  "extCodeHashCreatedAndDeletedAccountRecheckInOuterCall": {
+    "_info": {
+      "comment": "EXTCODEHASH/EXTCODESIZE of an account created then deleted in a CALL, checking results after the CALL returns"
+    },
+    "env": {
+      "currentCoinbase": "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+      "currentDifficulty": "0x20000",
+      "currentGasLimit": "1000000",
+      "currentNumber": "1",
+      "currentTimestamp": "1000",
+      "previousHash": "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+    },
+    "expect": [
+      {
+        "indexes": {
+          "data": -1,
+          "gas": -1,
+          "value": -1
+        },
+        "network": [
+          ">=Constantinople"
+        ],
+        "result": {
+          "deadbeef00000000000000000000000000000000": {
+            "storage": {
+              "0x00": "0x396daaf3a7871f857cc9b5878c6bb66c394a6d109706095617db9f4e4c22edcd",
+              "0x01": "0x03",
+              "0x02": "0x396daaf3a7871f857cc9b5878c6bb66c394a6d109706095617db9f4e4c22edcd",
+              "0x03": "0x03"
+            }
+          },
+          "deadbeef00000000000000000000000000000001": {
```

Comment:
I think it is better to add account address (MLOAD 0) here 
and check that it does not exit. this way we make sure that call to that account was successfull. 
```
"accountaddress": {
"shouldnotexist" : "1"  
}
```

Reply:
Added.